### PR TITLE
feat: Remove the PNS executor. Fixes #7804

### LINKED ISSUE #7804: Remove `pns` executor
👀 

Should be able to use Emissary.

Also, remove the `libcap` and `procps` libraries one complete.

### PR DESCRIPTION
**Pull request by alexec**
Signed-off-by: Alex Collins <alex_collins@intuit.com>

Fixes #7804

### CODE REVIEW

**Comment by alexec** [comment]
This PR will always wait for those checks, because the name changed when we removed the executor.

File: cmd/argoexec/commands/root.go [hunk 1]
```diff
@@ -76,11 +75,10 @@ func NewRootCommand() *cobra.Command {
 
 func initExecutor() *executor.WorkflowExecutor {
 	version := argo.GetVersion()
-	executorType := os.Getenv(common.EnvVarContainerRuntimeExecutor)
-	log.WithFields(log.Fields{"version": version.Version, "executorType": executorType}).Info("Starting Workflow Executor")
+	log.WithFields(log.Fields{"version": version.Version}).Info("Starting Workflow Executor")
 	config, err := clientConfig.ClientConfig()
 	checkErr(err)
-	config = restclient.AddUserAgent(config, fmt.Sprintf("argo-workflows/%s argo-executor/%s", version.Version, executorType))
+	config = restclient.AddUserAgent(config, fmt.Sprintf("argo-workflows/%s", version.Version))
```
Comment (terrytangyuan) [inline cmd/argoexec/commands/root.go:81]: I think we can just remove the `executorType` but keep `argo-executor` to be consistent with UAs from controller, server, etc.

**Review by terrytangyuan — COMMENTED** [review]
Just one minor comment. Otherwise LGTM!

File: cmd/argoexec/commands/root.go (hunk 1 above)
Comment (alexec) [reply cmd/argoexec/commands/root.go:81]: hard code to emissary?

Comment (terrytangyuan) [reply cmd/argoexec/commands/root.go:81]: Either `fmt.Sprintf("argo-workflows/%s emissary", version.Version)` or `fmt.Sprintf("argo-workflows/%s argo-executor", version.Version)` would work. The benefit of including a more general term like "executor" would give the operator a more meaningful signal when monitoring the traffic.

Comment (alexec) [reply cmd/argoexec/commands/root.go:81]: `argo-executor=not-applicable`?

Comment (alexec) [reply cmd/argoexec/commands/root.go:81]: So just `argo-executor` then?

Comment (terrytangyuan) [reply cmd/argoexec/commands/root.go:81]: Sounds good

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**